### ISSUE TITLE
Background transparency

### ISSUE THREAD

**Issue by gtrcoi** [oncomouse/lushwal.nvim#4]
Some discussion about transparency on the lush repo https://github.com/rktjmp/lush.nvim/issues/38

Editing `~/.config/nvim/colors/lushwal.vim` and changing `guibg=#[hexcolor]` on certain lines to `guibg=NONE` works to remove the background and use terminal opacity.

I'm unsure how to make lushwal do this, if anyone knows that would be awesome.

**Comment by MasserCapper67** [comment]
Partially fixed the problem.
If you go to "~/.local/share/nvim/lazy/lushwal.nvim/lua/lushwal/colors.lua" and delete / comment the line 39 (background = background,).
My problem is that since I'm using lazyvim, when I try to update my plugins I get an error message that tells me I've modified the local lushwal.nvim files, which is kind of annoying. 
Does anyone know a better approach to solve this issue?
![image](https://github.com/user-attachments/assets/70962047-b7d4-49f3-a032-f24853051681)

**Comment by oncomouse** [comment]
This should be fixed in f974fb8.

It's off by default, so you'll need to set `vim.g.lushwal_configuration.transparent_background` to `true.

Here's an example from my LazyVim configuration:

~~~lua
{
	"oncomouse/lushwal.nvim",
	cmd = { "LushwalCompile" },
	init = function()
		vim.g.lushwal_configuration = {
				transparent_background = true,
		}
	end,
	dependencies = {
		{ "rktjmp/lush.nvim" },
		{ "rktjmp/shipwright.nvim" },
	},
},
~~~

@MasserCapper67, for future reference, LazyVim draws from plugin github repos to make changes, so I would need to fix the problem on my mind (which I've done) or you could fork my repo and fix it for yourself. If you ever wanted to provide a plugin author with your solution, you [create something called a pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request).
